src/Anonymization/Anonymizer/Core/FloatAnonymizer.php

```diff
@@ -31,15 +31,20 @@ public function anonymize(Update $update): void
         $min = $this->options->get('min');
         $precision = 10 ** $this->options->get('precision', 2);
 
+        $randomInt = $this->getRandomIntExpression(
+            (int) \floor($max * $precision),
+            (int) \ceil($min * $precision)
```

Comment:
> `\floor()` and `\ceil()` will return int already

The PHP doc tells that both those functions return a float :

- https://www.php.net/manual/fr/function.floor.php
- https://www.php.net/manual/fr/function.ceil.php

> and that's actually why they are here

They were not here initially, I added those function calls.

> I'm also unsure about this change, did you experience any problems here ?

If one of `max` or `min` option is a float, an error occurs because `getRandomIntExpression()` expects integers as argument.

Reply:
I learnt something today, fun stuff, I never stumbled upon any crash because of this.